python/pyarrow/_parquet.pxd

```diff
@@ -495,6 +500,9 @@ cdef extern from "parquet/api/writer.h" namespace "parquet" nogil:
             Builder* disable_write_page_index()
             Builder* enable_page_checksum()
             Builder* disable_page_checksum()
+            Builder* enable_content_defined_chunking()
+            Builder* disable_content_defined_chunking()
+            Builder* content_defined_chunking_options(const CdcOptions options)
```

Comment:
```suggestion
            Builder* content_defined_chunking_options(CdcOptions options)
```

It seems that we can remove `const`?

Reply:
Right, cython headers are easy to forget. Removed.